cmd/minikube/cmd/config/validations.go

```diff
@@ -75,3 +76,11 @@ func IsValidPath(name string, path string) error {
 	}
 	return nil
 }
+
+func IsValidUrl(string, isoURL string) error {
```

Comment:
Yes it works with file based URI also https://golang.org/src/net/url/url_test.go  (check L#271) but it not identify OS specific rule like for windows file://<path> works and for Mac file:/<path> and linux file:///<path> works that something user should know and add it accordingly.

I will add a generic test case around it.